api/v1alpha1/jwt_types.go

```diff
@@ -54,7 +54,20 @@ type JWTProvider struct {
 	// The claim must be of type; string, int, double, bool. Array type claims are not supported
 	//
 	ClaimToHeaders []ClaimToHeader `json:"claimToHeaders,omitempty"`
-	// TODO: Add TBD JWT fields based on defined use cases.
+
+	// FromHeaders is a list of HTTP request headers to extract the JWT token from.
+	// By default JWT Authentication extract JWT token from the Authorization HTTP Header using Bearer schema or
+	// extract it from access_token query parameters.
+	// See: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/jwt_authn/v3/config.proto
+	FromHeaders []JWTFromHeader `json:"fromHeaders,omitempty"`
+
+	// FromCookies is a list of cookie names to extract the JWT token from.
+	// See: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/jwt_authn/v3/config.proto
+	FromCookies []string `json:"fromCookies,omitempty"`
```

Comment:
can we only implement `cookies` for now ? since that is the only use case you are interested in ?